force-app/main/default/lwc/apiSetterGetter/apiSetterGetter.css

```diff
@@ -1,12 +1,12 @@
-c-todo-list {
+.c-todo-list {
```

Comment:
@pchittum can you remove the `c-` namespace in the class name? It could be misleading if people miss the `.` class selector here.

Reply:
Sure thing.